components/brave_wallet_ui/common/hardware/types.ts

```diff
@@ -35,60 +34,171 @@ export type SignHardwareOperationResult = HardwareOperationResult & {
   payload?: EthereumSignedTx | FilSignedLotusMessage | Buffer | string
 }
 
-export enum LedgerDerivationPaths {
-  LedgerLive = 'ledger-live',
-  Legacy = 'legacy',
-  Deprecated = 'deprecated'
+export enum DerivationScheme {
+  EthLedgerLive = 'EthLedgerLive',
+  EthLedgerLegacy = 'EthLedgerLegacy',
+  EthLedgerDeprecated = 'EthLedgerDeprecated',
+  SolLedgerDefault = 'SolLedgerDefault',
+  SolLedgerLedgerLive = 'SolLedgerLedgerLive',
+  SolLedgerBip44Root = 'SolLedgerBip44Root',
+  FilLedgerMainnet = 'FilLedgerMainnet',
+  FilLedgerTestnet = 'FilLedgerTestnet',
+  BtcLedgerMainnet = 'BtcLedgerMainnet',
+  BtcLedgerTestnet = 'BtcLedgerTestnet',
+
+  EthTrezorDefault = 'EthTrezorDefault'
+}
```

Comment:
Can we use a `as const` object instead?
https://www.typescriptlang.org/docs/handbook/enums.html#objects-vs-enums